pallets/staking/src/lib.rs

```diff
@@ -327,6 +378,110 @@ pub mod pallet {
             Self::deposit_event(Event::Unstaked(sender, amount));
             Ok(().into())
         }
+
+        /// Relay chain accounts process the pending unstake by unbonding assets.
+        ///
+        /// May only be called from `T::WithdrawOrigin`.
+        ///
+        /// - `agent`: the multisig account of relay chain.
+        /// - `owner`: the account which performs `unstake` operation
+        /// - `amount`: the assets can be unbond for the owner's unstaking request.
+        #[pallet::weight(10_000)]
+        #[transactional]
+        pub fn process_pending_unstake(
+            origin: OriginFor<T>,
+            agent: T::AccountId,
+            owner: T::AccountId,
```

Comment:
Is the owner different from origin?

Reply:
The origin should be the one to one map to agent, need also to be multisig.